data/korath/korath missions.txt

```diff
@@ -257,3 +270,207 @@ mission "Small Scale Delivery"
+				to display
+					"reputation: Hai (Unfettered)" < 30
+			`	Batu looks less satisfied. "We've found only a few dozen items, and there aren't any live Korath - or even Korath bodies. With all the fighters being disabled, we'd expect to find a bit more. This is likely part of an operation to pillage and sell Korath outfits. Worse, they may even be trading with the Korath. The operation's Stormhold facilities were for showcasing and auctioning Korath goods. By taking their sales center, we may have shut down the operation."`
+
+			label questions
+			choice
+				`	"Who are the Korath?"`
+					to display
+						not "know who the korath are"
+						"reputation: Korath" > -100
+						not "asked choot'k about the korath"
+				`	"I've fought the Korath for a long time, but I don't really know who they are."`
+					to display
+						not "know who the korath are"
+						"reputation: Korath" <= -100
+						not "asked choot'k about the korath"
+				`	"What can you tell me about the Korath?"`
+					to display
+						not "asked choot'k about the korath"
+						or
+							has "know who the korath are"
+							"reputation: Korath" <= -100
+				`	"Why are there so many Korath artifacts?"`
+					goto somany
+			action
+				set "know who the korath are"
+				set "asked choot'k about the korath"
+			`	Choot'k responds. "The Korath are a group of lizards that lived south of Hai space until a few centuries ago. They used to be obsessed with art, music, and other esoterics. I care more for practical matters, but I must admit that I enjoy our old recordings of their music. Haunting guttural hums accented by angelic vocals with lyrics in their intricate language. The mood goes from eerie and hopeful to enlivening to frightening from verse to verse and draws the listener with them. Accompanied by mesmerizing visuals, the experience absorbs you in alluring totality."`
+			`	He looks slightly embarrassed and switches to practical matters. "The Hai and Korath were on good terms - trading goods and culture - until the lizards got desperate for resources and invaded us. Their technology is far beyond ours, but we defended valiantly. I'm not sure we would have won on our own, but they nearly wiped themselves out in bloody infighting. Ultimately, it was stronger and older races that ended a self-destructive spiral towards extinction. As an Unfettered, I have grievances with the Drak, but I am glad those space cockroaches saved the Korath from themselves.`
```

Comment:
```suggestion
			`	He looks slightly embarrassed and switches back to the subject. "The Hai and Korath were on good terms - trading goods and culture - until the lizards got desperate for resources and invaded us. Their technology is far beyond ours, but we defended valiantly. I'm not sure we would have won on our own, but they nearly wiped themselves out in bloody infighting. Ultimately, it was stronger and older races that ended a self-destructive spiral towards extinction. As an Unfettered, I have grievances with the Drak, but I am glad those space cockroaches saved the Korath from themselves.`
```

Reply:
This odd wording refers back to the previous paragraph where he claims he, "cares more for practical matters," while implying that may not be 100% true.